quint/test/runtime/compile.test.ts

```diff
@@ -1015,7 +1022,16 @@ describe('compiling specs to runtime values', () => {
         |run run1 = (n' = 1).then((n' = n + 1).repeated(3))
         `)
 
-      assertVarAfterCall('n', '4', input)
+      assertVarAfterCall('n', '4', 'run1', input)
+    })
+
+    it('reps', () => {
+      const input = dedent(
+        `var n: int
+        |run run1 = (n' = 1).then(3.reps(i => n' = n + 1))
```

Comment:
```suggestion
        |run run1 = (n' = 1).then(3.reps(_ => n' = n + 1))
```

Reply:
Also: can you add a test where `i` is actually used?

Reply:
Yep, added